index.html

```diff
@@ -424,68 +427,72 @@ <h3 id="webidl-conform">WebIDL Conformance</h3>
         title="examples">example</a> for sample code demonstrating how to fire
         an untrusted touch event.
       </p>
-
-      <dl title='[Constructor(DOMString type, optional TouchEventInit eventInitDict)] interface TouchEvent : UIEvent' class='idl' data-merge='TouchEventInit'>
-        <dt>readonly attribute <a><code>TouchList</code></a> touches</dt>
+      <pre class="idl">
+dictionary TouchEventInit : EventModifierInit {
+             sequence<Touch> touches = [];
```

Comment:
I believe you'll need to use &gt; and &lt; here for the angle brackets (they're not showing up in the text).